integration-tests/helpers/index.js

```diff
@@ -635,14 +640,8 @@ function setShouldKill (value) {
   })
 }
 
-// @ts-expect-error assert.partialDeepStrictEqual does not exist on older Node.js versions
-// eslint-disable-next-line n/no-unsupported-features/node-builtins
-const assertObjectContains = assert.partialDeepStrictEqual || function assertObjectContains (actual, expected, msg) {
```

Comment:
While I understand the wish for asymmetric matchers, this is less than ideal. The output of partialDeepStrictEqual in error cases is just vastly better than the fallback. As such, I think we should not use this by default.

If we want to use these, right away (I do plan on implementing these in Node.js assert soon as well), than we should run the native method first and only in the failure case run the fallback with asymmetric matchers.

Reply:
done